docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,466 @@
+but creating this configuration now means the system is preconfigured.
+
+Save this file as `collectorconfig.yaml`:
+
+```yaml
+---
+apiVersion: options.keptn.sh/v1alpha1
+kind: KeptnConfig
+metadata:
+  name: keptnconfig-sample
+  namespace: keptn-lifecycle-toolkit-system
+spec:
+  OTelCollectorUrl: 'jaeger-collector.keptn-lifecycle-toolkit-system.svc.cluster.local:4317'
+  keptnAppCreationRequestTimeoutSeconds: 30
+```
+
+Apply the file and restart KLT to pick up the new config:
+
+```shell
+kubectl apply -f collectorconfig.yaml
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator --watch
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l component=scheduler
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l component=scheduler --watch
+```
+
+## Create Namespace for Demo Application
+
+Save this file as `namespace.yaml`.
+The annotation means that Keptn Lifecycle Toolkit is active for workloads in this namespace.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: keptndemo
+  annotations:
+    keptn.sh/lifecycle-toolkit: enabled
+```
+
+Create the namespace:
+
+```shell
+kubectl apply -f namespace.yaml
+```
+
+## Deploy Demo Application
+
+It is time to deploy the demo application.
+
+Save this manifest as `app.yaml`:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: keptndemo
+  labels:
+    app.kubernetes.io/name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: nginx
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: keptndemoapp
+        app.kubernetes.io/name: nginx
+        app.kubernetes.io/version: 0.0.1
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
+        ports:
+        - containerPort: 80
+---
+apiVersion: apps/v1
+kind: Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  namespace: keptndemo
+spec:
+  selector:
+    app.kubernetes.io/name: nginx
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 80
+```
+
+Now apply:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+Keptn looks for these 3 labels:
+
+- `app.kubernetes.io/part-of`
+- `app.kubernetes.io/name`
+- `app.kubernetes.io/version`
+
+These are [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)
+but if you want to use different labels, you can swap for:
+
+- `keptn.sh/app` instead of `app.kubernetes.io/part-of`
+- `keptn.sh/workload` instead of `app.kubernetes.io/name`
+- `keptn.sh/version` instead of `app.kubernetes.io/version`
+
+## Explore Keptn
+
+Keptn is now aware of your deployments and is generating DORA statistics about them.
+
+Keptn has created a CRD to track your application.
```

Comment:
```suggestion
Keptn has created a resource to track your application.
```
Flo corrected me about terminology:  A CRD is the definition itself; in other words, the syntax.  Each instance that is created using that CRD is just a CR or resource.  All the Keptn specific resources are CRs but the pure Kubernetes resources are not "Custom" so just resources.  So I'm just calling them all resources.